docs/node-operators/get-started/setup-datanode.md

```diff
@@ -293,6 +309,21 @@ To start Vega, run the following command:
 vega start --home="YOUR_VEGA_HOME_PATH" --tendermint-home="YOUR_TENDERMINT_HOME_PATH"
 ```
 
+## Fetch network history
+After starting a data node, you can load in a segment of network history, if you want your node to have more data than provided by the current height. This is particularly useful if you're running an archive node.
+
+To see how much network history your data node has, run the following command:
+
+```shell
+vega datanode network-history show --home="YOUR_DATA_NODE_HOME_PATH"
+```
+
+To fetch a network history segment, run the command below. Use the ID of the segment you want (for example, the oldest) and load all data from a block height. `500` is used as an example.
```

Comment:
```suggestion
To fetch a network history segment, run the command below. Use the ID of the segment you want (for example, the oldest) followed by the number of blocks prior to the segments height that you want fetch. `2000` is used in the following example.  This will result in all blocks from height 3000 to 5000 being retrieved.
```